runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -158,9 +158,9 @@ void setConfigKeyValues(const CfgKeyValues &keyValues)
 } // namespace
 
 nnfw_session::nnfw_session()
-    : _subgraphs{nullptr}, _execution{nullptr},
-      _kernel_registry{std::make_shared<onert::frontend::custom::KernelRegistry>()},
-      _tracing_ctx{nullptr}
+  : _subgraphs(nullptr), _execution(nullptr),
+    _kernel_registry(std::make_shared<onert::frontend::custom::KernelRegistry>()),
+    _tracing_ctx(nullptr)
```

Comment:
Change to brace because of unexpected format break:

```
nnfw_session::nnfw_session()
  : _subgraphs{nullptr}, _execution{nullptr},
    _kernel_registry{std::make_shared<onert::frontend::custom::KernelRegistry>()}, _tracing_ctx{
                                                                                     nullptr}
```

Reply:
@hseok-oh Not important note: I took a look since brace is more likely to make unexpected format. You seemed to change to parenthesis, not brace.

Reply:
Does this mean we don't allow curly brace initializers?

Reply:
>I took a look since brace is more likely to make unexpected format. You seemed to change to parenthesis, not brace.

Right. `Change to brace` -> `Change brace`

Reply:
>Does this mean we don't allow curly brace initializers?

No. It's just one choice for formatting. If you don't want to change this, we can remain above comment's change, or use clang-format off/on.

Reply:
I would prefer keeping the current implementation without clang format on/off comments.

- Brace-initializer and parenthesis-initializer are not always interchangeable.
- We have a lot of code like this

Reply:
It's strange that clang-format emits "unexpected format" error. 😕

Reply:
@hseok-oh 

I see the formatting of updated commit. Is it the one you've mentioned "_unexpected format_"?

![image](https://user-images.githubusercontent.com/12553213/101318278-6c232780-38a3-11eb-92d7-6ccfa2264e2b.png)

Though the previous one looks better, I don't mind the current one.

Reply:
>Is it the one you've mentioned "unexpected format"?

Right